### PR TITLE
do not impose limits on unmetered TXes

### PR DESCRIPTION
**Pull request by oxade**
We don't want to limit system/unmetered TXes
we also increase the limits to 2048

### CODE REVIEW

**Comment by oxade** [comment]
@emmazzz how do we actually test this?

File: crates/sui-adapter/src/adapter.rs [hunk 1]
```diff
@@ -90,9 +90,14 @@ pub fn new_session<
     vm: &'v MoveVM,
     state_view: &'r S,
     input_objects: BTreeMap<ObjectID, (/* by_value */ bool, Owner)>,
+    metered: bool,
```
Comment (tnowacki) [inline crates/sui-adapter/src/adapter.rs:93]: nit `is_metered`

File: crates/sui-cost-tables/src/bytecode_tables.rs [hunk 2]
```diff
@@ -68,6 +68,10 @@ impl<'a> GasStatus<'a> {
         }
     }
 
+    pub fn metered(&self) -> bool {
```
Comment (tnowacki) [inline crates/sui-cost-tables/src/bytecode_tables.rs:71]: nit is_metered

File: crates/sui-framework/src/natives/object_runtime/mod.rs [hunk 3]
```diff
@@ -84,6 +84,8 @@ pub struct ObjectRuntime<'a> {
     pub(crate) test_inventories: TestInventories,
     // the internal state
     pub(crate) state: ObjectRuntimeState,
+    // whether or not this TX is gas metered
+    metered: bool,
```
Comment (tnowacki) [inline crates/sui-framework/src/natives/object_runtime/mod.rs:88]: is_metered

**Review by tnowacki — APPROVED** [review]

**Pull request merged**